.github/workflows/slow-tests.yml

```diff
@@ -0,0 +1,67 @@
+# A weekly workflow to run slow tests with MLFLOW_RUN_SLOW_TESTS environment variable set to true.
+name: MLflow Slow Tests
+
+on:
+  pull_request:
+    paths:
+      # Run this workflow in PR when relevant files change
+      - '.github/workflows/slow-tests.yml'
+      - 'tests/pyfunc/docker/test_docker.py'
+  workflow_dispatch:
```

Comment:
Per my understanding it is used for manual run right? Since this is a bit heavy load job (and need ML many libraries to be installed), it would be useful to be able to run it on test infra rather than our machine.

Reply:
makes sense, let's keep it